### ISSUE TITLE
Complete list of HRP countries

### ISSUE THREAD

**Issue by turnerm** [OCHA-DAP/pa-aa-toolbox#148]
Perhaps we can get DPT to support?

**Comment by caldwellst** [comment]
Yeah, as I suggested on Slack, I think a hackathon could be great. Probably best to wait until we have our final 0.5 data sources ready, and then clear understanding of what each config file will need.
